src/core/features/settings/pages/site/site.html

```diff
@@ -18,13 +18,16 @@ <h1>{{ 'core.settings.preferences' | translate}}</h1>
         <core-loading [hideUntil]="handlers.loaded">
             <ion-list>
                 <ion-item *ngFor="let handler of handlers.items" [ngClass]="['core-settings-handler', handler.class]"
-                    [attr.aria-label]="handler.title | translate" detail="true" (click)="handlers.select(handler)" button
-                    [attr.aria-current]="handlers.getItemAriaCurrent(handler)">
+                    [attr.aria-label]="handler.title | translate" (click)="handlers.select(handler)" [button]="!handler.toggle"
+                    [detail]="!handler.toggle" [attr.aria-current]="handlers.getItemAriaCurrent(handler)">
                     <ion-icon [name]="handler.icon" slot="start" *ngIf="handler.icon" aria-hidden="true">
                     </ion-icon>
                     <ion-label>
                         <p class="item-heading">{{ handler.title | translate}}</p>
                     </ion-label>
+                    <ion-toggle *ngIf="handler.toggleChecked !== undefined" [(ngModel)]="handler.toggleChecked"
```

Comment:
```suggestion
                    <ion-toggle *ngIf="handler.toggle" [(ngModel)]="handler.toggleChecked"
```

This should be more readable and we're already using the same logic on the lines above.